conans/client/command.py

```diff
@@ -1846,6 +1898,10 @@ def _add_common_install_arguments(parser, build_help):
                              '-s compiler=gcc')
     parser.add_argument("-u", "--update", action='store_true', default=False,
                         help="Check updates exist from upstream remotes")
+    parser.add_argument("-if", "--install-folder", action=OnceArgument,
```

Comment:
This is producing the `conan test` to get the `-if` and I think it is useless there.

Reply:
Yes, removing